recipes/shiori/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+name: Shiori
+description: Shiori is a simple bookmarks manager written in the Go language. Intended as a simple clone of Pocket. You can use it as a command line application or as a web application. This application is distributed as a single binary, which means it can be installed and used easily.
+section: net
```

Comment:
I think `web` is the better choice here.

> web – Best for general web-based applications and tools.
net – Suitable if the application primarily deals with networking (e.g., web servers, proxies).